FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -154,20 +154,21 @@ NS_SWIFT_NAME(RemoteConfig)
 /// Returns the FIRRemoteConfig instance configured for the default Firebase app. This singleton
 /// object contains the complete set of Remote Config parameter values available to the app,
 /// including the Active Config and Default Config. This object also caches values fetched from the
-/// Remote Config Server until they are copied to the Active Config by calling activateFetched. When
-/// you fetch values from the Remote Config Server using the default Firebase namespace service, you
-/// should use this class method to create a shared instance of the FIRRemoteConfig object to ensure
-/// that your app will function properly with the Remote Config Server and the Firebase service.
+/// Remote Config Server until they are copied to the Active Config by calling
+/// `activateWithCompletion`. When you fetch values from the Remote Config Server using the default
+/// Firebase namespace service, you should use this class method to create a shared instance of the
+/// FIRRemoteConfig object to ensure that your app will function properly with the Remote Config
+/// Server and the Firebase service.
 + (nonnull FIRRemoteConfig *)remoteConfig NS_SWIFT_NAME(remoteConfig());
 
 /// Returns the FIRRemoteConfig instance for your (non-default) Firebase appID. Note that Firebase
 /// analytics does not work for non-default app instances. This singleton object contains the
 /// complete set of Remote Config parameter values available to the app, including the Active Config
 /// and Default Config. This object also caches values fetched from the Remote Config Server until
-/// they are copied to the Active Config by calling activateFetched. When you fetch values from the
-/// Remote Config Server using the default Firebase namespace service, you should use this class
-/// method to create a shared instance of the FIRRemoteConfig object to ensure that your app will
-/// function properly with the Remote Config Server and the Firebase service.
+/// they are copied to the Active Config by calling `activateWithCompletion`. When you fetch values
```

Comment:
nit: `activateWithCompletion:` and the same for the followings.

Reply:
updated